instagram/src/App.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from "react";
+import PostPage from "./components/PostContainer/PostPage";
+import authenticateHOC from "./components/Authentication/Authenticate";
+import LoginPage from "./components/Login/Login";
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {};
+  }
+  render() {
+    return (
+      <div className="App">
+        <PostPage />
```

Comment:
I would try to extract the header component and render it here.  As you've built it there's no reason for App.js to be your top level component, you could just make PostPage your top level component.